migration/validator-activation.md

```diff
@@ -0,0 +1,76 @@
+
+## Validator Activation Tool
+
+This guide covers **Phase 3: Validator Activation**, which starts on **November 19th**. The Validator Activation Tool facilitates the transition from the PoW chain to the PoS chain. We recommend running the tool before the activation window begins to allow time for database synchronization, as this process can take some time.
+
+| **Phase** | **Start Date** | **End Date** |
+| --- | --- | --- |
+| Validator Registration | 12th September | 6th October |
+| Pre-Stake Phase | 7th October | 10th November |
+| **Validator Activation** | **19th November** | - |
+
+The steps outlined in this guide are only applicable during the **Validator Activation Phase**.
+
+### Prerequisites
+- **Validator Registration**: You must have already [registered as a validator](validator-registration)
+- **PoW Full Node**: You need access to a fully synchronized Nimiq PoW full node with RPC access enabled. If you are not already running a full node, follow the instructions in the [core-js repository](https://github.com/nimiq/core-js) to set it up and sync with the main network
+    - If you are running a PoW client, ensure it is fully synced and configured properly
+    - Use this [sample configuration file](https://github.com/nimiq/core-js/blob/master/clients/nodejs/sample.conf) to enable RPC access on your PoW client
+    - During this phase, you will need to run both the PoW and PoS clients. Ensure they are configured to run on **different RPC ports** to avoid conflicts if you are running both clients on the same machine. Alternatively, running them on separate servers is also possible.
+- **PoS Blockchain Repository**: Clone the [PoS blockchain repository](https://github.com/nimiq/core-rs-albatross?tab=readme-ov-file#installation) following the installation instructions
+
+### **Step 1: Add Validator Data into the Configuration File**
+
+Once your PoS client is set up, follow these steps:
+
+1. Check if the `$HOME/.nimiq` folder exists:
+    1. If it exists, open the configuration file in `$HOME/.nimiq/client.toml` and locate the dedicated section for validators at the end
+    2. If it doesn’t exist, create the `$HOME/.nimiq` folder and inside, create a `client.toml` file based on this [example](https://github.com/nimiq/core-rs-albatross/blob/albatross/lib/src/config/config_file/client.example.toml)
+2. Paste your **validator data** into the validator section (public key, signing key, voting key)
+3. Set the **network** is set to `main-albatross` and the `sync_mode` to `full` or `history`. Only full or history nodes are eligible to become validators
```

Comment:
Theoretically the tool can also be run in `light` mode. But validators must use only these two mode as you say